changelog/5355.breaking.rst

```diff
@@ -0,0 +1 @@
+The default response format from the VSO client is now a table.
```

Comment:
Shouldn't this be a removal? didn't we deprecate the previous default?

Reply:
We did, this should removal.